schema/test/unit/computed_columns/form_change_payment_percentage_test.sql

```diff
@@ -93,6 +111,34 @@ select is(
   'should return 200 if the form change does not have adjustedEmissionsIntensityPerformance and has all information to calculate ei performance'
 );
 
+-- should return 100 if payment percentage is very large (over 100%)
+select is(
+  (
+    with record as (
+    select row(form_change.*)::cif.form_change
+    from cif.form_change where id=4
+    ) select cif.form_change_payment_percentage((select * from record))
+  ),
+  (
+    100::numeric
+  ),
+  'payments percentage is 100 from the formula: 100 - ((-1.5) x 500 + 145) = 705 > 100 ==> return 100'
+);
+
+-- should return 0 if payment percentage is negative
+select is(
+  (
+    with record as (
+    select row(form_change.*)::cif.form_change
+    from cif.form_change where id=5
+    ) select cif.form_change_payment_percentage((select * from record))
+  ),
+  (
+    0::numeric
+  ),
+  'payments percentage is 0 from the formula: 100 - ((-1.5) x 50 + 145) = -1245 < 0 ==> return 0'
+);
+
 -- 200.00,
```

Comment:
leftover?

Reply:
Included this comment to be consistent with line 85 of the same file:
`'payments percentage is 30 from the formula: 100 - ((-1.5) x 50 + 145) = 30'`
Can certainly remove if it's not useful!

Reply:
I meant the `-- 200.00` comment floating at the end of the test here